app/web/service/search.ts

```diff
@@ -32,7 +32,7 @@ export async function userSearch(
   const req = new UserSearchReq();
   req.setPageToken(pageToken);
 
-  if (query !== "") {
+  if (query !== undefined && query !== "") {
```

Comment:
Looks like you can just do:

```javascript
if (query) {}
```

https://developer.mozilla.org/en-US/docs/Glossary/Falsy

<img width="353" alt="Screenshot 2024-09-17 at 9 29 10 AM" src="https://github.com/user-attachments/assets/92df5c83-8e2a-4eef-9bc1-053e2743cad9">

<img width="345" alt="Screenshot 2024-09-17 at 9 29 15 AM" src="https://github.com/user-attachments/assets/e98946aa-ed84-4b90-8d55-dcc80f8b5a2b">

Reply:
cool!!!

Reply:
updated ✅

Reply:
I've spend to much time learning these things on PHP that I forgot to learn it about JS